packages/react-admin-date-picker/src/DateRangePicker.tsx

```diff
@@ -31,7 +31,9 @@ export const FinalFormDateRangePicker: React.FC<IProps> = ({
     endPlaceholder,
     ...props
 }) => {
-    const localeName = useLocaleName();
+    const intl = useIntl();
+    const localeName = intl.locale;
```

Comment:
useLocaleName() is interchangeable with intl.locale!
I am impressed, this is some neat refactoring.